react/clients/cse.ts

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios'
+import { CSE_CX, CSE_KEY } from '../utils/constants'
+
+export const cseSearch = async (query: string) => {
+  const res = await axios(`
+     https://www.googleapis.com/customsearch/v1?q=${query}&cx=${CSE_CX}&key=${CSE_KEY}
+  `)
+  const results = await res.data.items
+  return results
+}
```

Comment:
I think it's a better idea to move this to `docs-graphql`

Reply:
I can do that, but do you have a reason why it is better to use graphql in this case?